src/balancereader/src/main/java/anthos/samples/financedemo/balancereader/BalanceReaderController.java

```diff
@@ -100,18 +101,18 @@ public BalanceReaderController(LedgerReader reader,
             Algorithm algorithm = Algorithm.RSA256(publicKey, null);
             this.verifier = JWT.require(algorithm).build();
         } catch (IOException
-                | NoSuchAlgorithmException
-                | InvalidKeySpecException e) {
-            LOGGER.severe(e.toString());
+            | NoSuchAlgorithmException
+            | InvalidKeySpecException e) {
+            LOGGER.fatal(e.toString());
             System.exit(1);
         }
         // Initialize cache
         CacheLoader loader =  new CacheLoader<String, Long>() {
             @Override
             public Long load(String accountId)
-                    throws ResourceAccessException,
-                           DataAccessResourceFailureException {
-                LOGGER.fine("loaded from db");
+                throws ResourceAccessException,
+                DataAccessResourceFailureException {
+                LOGGER.debug("Cache loaded from db");
```

Comment:
This could probably be bumped up to 'info'

This is a signal into the current state of the program, and it only happens once on startup.  (Log files flooded with INFO messages from repeated operations are hard to debug)